Doc/library/glob.rst

```diff
@@ -77,6 +77,10 @@ For example, ``'[?]'`` matches the character ``'?'``.
       Using the "``**``" pattern in large directory trees may consume
       an inordinate amount of time.
 
+   .. note::
```

Comment:
I would suggest could rather than might. The former suggests possibility, but the latter also implies a degree of indeterminacy, that something could happen but also that it would be hard to predict it. (Applies to other instances of the same working in the pull request too.)

Reply:
Makes sense, thanks!